4480 sns Workflow (#4767)
awsdocs/aws-doc-sdk-examples: cpp/example_code/sqs/hello_sqs/README.md

# Hello SQS for the SDK for C++

## Overview

> This folder provides a CMake "Hello SQS" project which uses the AWS SDK of C++ to call the Amazon Simple Queue Service (Amazon SQS).

Reviewer: ```suggestion
This folder provides a CMake "Hello SQS" project that uses the AWS SDK for C++ to call Amazon Simple Queue Service (Amazon SQS).
```

> This folder provides a CMake "Hello SQS" project which uses the AWS SDK of C++ to call the Amazon Simple Queue Service (Amazon SQS).

Reviewer: ```suggestion
This folder provides a CMake "Hello SQS" project that uses the AWS SDK for C++ to call Amazon Simple Queue Service (Amazon SQS).
```

## ⚠ Important

* Running this code might result in charges to your AWS account.
* Running the tests might result in charges to your AWS account.
* We recommend that you grant your code least privilege. At most, grant only the minimum permissions required to perform the task. For more information, see [Grant least privilege](https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#grant-least-privilege).
* This code is not tested in every AWS Region. For more information, see [AWS Regional Services](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services).

## Run the Hello SQS App

### Prerequisites

Before using this example, first complete the installation and setup steps
for [Getting started](https://docs.aws.amazon.com/sdk-for-cpp/v1/developer-guide/getting-started.html) in the AWS SDK for
C++ Developer Guide.

### Instructions

This example uses the CMake build system. For information about the CMake build system, see https://cmake.org/.

Many Integrated Development Environments (IDEs) support CMake. If your preferred IDE supports CMake, follow the IDE's instructions to open this CMake project.

> This project can also be built from a command line interface using the following commands.

Reviewer: ```suggestion
You can also build this project from a command line interface using the following commands.
```

```sh
mkdir build 
cd build
cmake --build ..
```

The built executable is named `hello_sqs`.

> This example can be run with the AWS Managed Policy "arn:aws:iam::aws:policy/AmazonSQSReadOnlyAccess".

Reviewer: ```suggestion
You can run this example with the AWS managed policy "arn:aws:iam::aws:policy/AmazonSQSReadOnlyAccess".
```

You can also use CMake to generate the input files for your native build system.
For more information, see https://cmake.org/cmake/help/latest/manual/cmake-generators.7.html.

> The [CMakeLists.txt](CMakeLists.txt) file contains the build settings. If your build is failing (particularly on Windows), you may need to modify this file.

Reviewer: ```suggestion
The [CMakeLists.txt](CMakeLists.txt) file contains the build settings. If your build is failing (particularly on Windows), you might need to modify this file.
```
